Bluetooth: mesh: Update default values for transport SAR configuration

akredalen opened the pull request:
Updated SAR default values to align with the latest mesh 1.1 drafts,
and relevant Bsim tests.

alxelax commented:
Hi @akredalen, try to rebase your PR. Seems the current issue is not related to your changes.

akredalen committed:
Bluetooth: mesh: Update default values for transport SAR configuration

Updated SAR default values to align with the latest mesh 1.1 drafts,
and relevant Bsim tests.

Signed-off-by: Stine Åkredalen <stine.akredalen@nordicsemi.no>
